fili-core/src/main/java/com/yahoo/bard/webservice/table/PermissiveConcretePhysicalTable.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.table;
+
+import com.yahoo.bard.webservice.data.config.names.TableName;
+import com.yahoo.bard.webservice.data.time.ZonedTimeGrain;
+import com.yahoo.bard.webservice.table.availability.Availability;
+
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * An instance of permissive Physical table that is backed by a single fact table.
```

Comment:
Unless a class is set up as a singleton, it isn't really an _instance_. It may be an _implementation_, which is perhaps what you meant?

Since this is a specialization of a `ConcretePhysicalTable`, the JavaDoc on this class should talk about what is different about this implementation. _Why_ does this class exist, what is it for, what concepts does it add on top of what `ConcretePhysicalTable` already does.

To put it a different way, merely adding (essentially) the class name to the javadoc of the parent class doesn't really make this a valuable / useful comment. 😄